jbtearlyaccesstarget/multiple/jbtearlyaccess-multiple.target

```diff
@@ -11,6 +11,12 @@
       <unit id="angularjs-eclipse-feature.feature.group" version="1.1.0.201511091212"/>
     </location>
 
+    <!-- Windup Migration -->
+    <location includeAllPlatforms="false" includeMode="slicer" includeSource="false" type="InstallableUnit">
+      <repository location="http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/"/>
+      <unit id="org.jboss.tools.windup.feature.feature.group" version="3.1.0.Alpha"/>
```

Comment:
This should be changed to 3.0.0.Alpha2 once those bits are available. ... Unless you're using my other suggestion above to omit this and simply link plugin.xml directly to http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/ , bypassing the need for a target platform definition.